src/netcdf_writer.jl

```diff
@@ -142,6 +142,19 @@ function NetCDFWriter(
         )
     end
 
+    if is_horizontal_space
+        interpolated_physical_z = []
```

Comment:
Let's split off this change to a new PR and have it merged. This is a bug in ClimaDiagnostics that might affect ClimaAtmos too

Reply:
ok I'll make a new PR now and add you as reviewer